kong/cmd/utils/log.lua

```diff
@@ -24,6 +25,15 @@ function _M.set_lvl(lvl)
   end
 end
 
+function _M.disable()
+  old_lvl = log_lvl
```

Comment:
point is; if someone messes up and forgets to reenable logging, the whole thing goes silent, and it'll be hard to track down any issues